docker/Dockerfile

```diff
@@ -5,14 +5,15 @@ WORKDIR /app
 COPY . .
 
 RUN sbt update compile stage
+RUN chmod a+rx /app/target/universal/stage/bin/*
 
 FROM openjdk:11-jre-slim
 
 WORKDIR /app
 
 COPY --from=build /app/target/universal/stage /app
 
-COPY docker/entrypoint.sh /app/entrypoint.sh
-RUN sed -i 's/\r$//' /app/entrypoint.sh && chmod +x /app/entrypoint.sh
+COPY --chmod=755 docker/entrypoint.sh /app/entrypoint.sh
+RUN sed -i 's/\r$//' /app/entrypoint.sh
```

Comment:
I would omit this sed line by setting LF mode with .gitattributes. wdyt?

Reply:
Oh yeah if you can get that working, should be cleaner than setting it here I agree

Reply:
Will submit separate PR for that then.